src/vivarium_census_prl_synth_pop/tools/make_results.py

```diff
@@ -334,13 +335,7 @@ def perform_post_processing(
         logger.info(f"Writing final results for {observer}.")
         obs_dir = build_output_dir(final_output_dir, subdir=observer)
         seed_ext = f"_{seed}" if seed != "" else ""
-        obs_data.to_hdf(
-            obs_dir / f"{observer}{seed_ext}.hdf",
-            "data",
-            format="table",
-            complib="bzip2",
-            complevel=9,
-        )
+        write_to_disk(obs_data.copy(), obs_dir / f"{observer}{seed_ext}.hdf")
```

Comment:
Seems like the .copy() should be moved into `write_to_disk`.

Reply:
We don't need to copy before write to disk in the observers. The copy was just to avoid the `SettingWithCopyWarning` which doesn't apply there.